packages/manager/apps/web/client/app/hosting/database/order/public/components/steps/db-categories-offers/controller.js

```diff
@@ -17,7 +17,14 @@ export default class WebHostingDatabaseOrderComponentsDbCategoriesOffersControll
   }
 
   getDbEngines() {
-    return this.model.dbCategory?.selectVersion?.engines || [];
+    return this.model.dbCategory?.selectVersion?.engines?.map((engine) => {
+      engine.versions.sort((a, b) => {
```

Comment:
I think you forgot a return here, also if engines is not set, getDbEngines will return undefined and not [] as previously

Reply:
Oh yes i forgot something here ! OK i will add it.